torch/csrc/jit/codegen/cuda/lower_expr_sort.h

```diff
@@ -7,9 +7,9 @@ namespace jit {
 namespace fuser {
 namespace cuda {
 
-std::vector<Expr*> reorderExprsForComputeAt(const std::vector<Expr*>& exprs);
+TORCH_CUDA_API std::vector<Expr*> reorderExprsTest();
```

Comment:
Is there any reason to name this function `-Test`?

Reply:
No of course not, just forgot to rename it.

Reply:
Reverted name.